apps/vs-agent/README.md

```diff
@@ -112,7 +112,7 @@ VS-A fetches capabilities from the `discovery.json` file (which is located at at
 
 ### Self VTR
 
-If the `SELF_VTR_ENABLED` environment variable is set to `true`, VS-A will enable features related to the Verifiable Trust Registry ([Self VTR Configuration Guide](./doc/self-vtr-routes.md)).
+If the `TESTVTR_ENABLE` environment variable is set to `true`, VS-A will enable features related to the Verifiable Trust Registry ([Self VTR Configuration Guide](./doc/self-vtr-routes.md)).
```

Comment:
Add some explanation about the variables and their default value. And a note regarding the org/service name and logo that will be the ones from AGENT_LABEL and AGENT_INVITATION_IMAGE_URL respectively